client/src/components/QA/AnswerContainer.jsx

```diff
@@ -12,8 +12,13 @@ const AnswerContainer = (props) => {
     <div className="answer-container">
       <div className="answer" key={props.body} onClick={(e) => { props.handleTrackingClick(e, e.currentTarget.className, 'Questions & Answers'); }}><b className="answer-body">A:</b>&nbsp;<span className="answer-text">{props.body}</span></div>
       <div className="answer-photos">{props.photos.map((photo, i) => {
+        let fullRes = photo;
+
+        let splitURL = photo.split('&w=');
```

Comment:
Good idea to just handle the url here in the map function. I always make things more complicated than they are.